src/pilot/assign_role.py

```diff
@@ -42,6 +41,12 @@
 import requests.packages
 from ironicclient.common.apiclient.exceptions import InternalServerError
 
+discover_nodes_path = os.path.join(os.path.expanduser('~'),
+                                   'pilot/discover_nodes')
+
+sys.path.append(discover_nodes_path)
+
+from discover_nodes.dracclient.client import DRACClient  # noqa
```

Comment:
Let's stick to - 
from discover_nodes.dracclient import client  
to be consistent with other imports.

Reply:
Can you explain why we are making the change so that assign_role uses the discover_nodes DRACClient instead of the DRACClient from python-dracclient?  Ideally, we want to eliminate the discover_nodes DRACClient and switch everything to the python-dracclient DRACClient.

Reply:
My intention behind this PR was to remove extra added dracclients (which was added in https://github.com/dsp-jetpack/JetPack/pull/202) and to make fewer changes in code.

In assign_role.py 

I have removed upstream dracclient and reverted back downstream dracclient (Same as Master).
https://github.com/dsp-jetpack/JetPack/blob/8cff5fd7c2ed90876d4cf851862a30bf756480d7/src/pilot/assign_role.py#L49

Above changes eventually point to 
https://github.com/dsp-jetpack/JetPack/blob/8cff5fd7c2ed90876d4cf851862a30bf756480d7/src/pilot/assign_role.py#L904

Earlier this change is done via newly imported upstream dracclient, but now it is working from downstream via inheritance.

Again this is up to us if we want to incorporate these changes.